do.py

```diff
@@ -112,19 +116,26 @@ def init():
 
 
 def lint():
-    paths = [pkg()[0], "tests", "setup.py", "do.py"]
+    paths = [
+        pkg()[0],
+        "art/sanity/sanity.py",
```

Comment:
It looks like we are covering all python files. In that case we can provide root directory

Reply:
sanity directory has pb2 files which are auto-generated. Touching them would create problem and other files are json and .proto. IMO, generated file is enough to check for linting